tests/github_app_test.py

```diff
@@ -0,0 +1,72 @@
+from typing import cast
+from unittest import mock
+
+import pytest
+
+from github import Auth
+
+from awx_plugins.credentials.github_app import github_app_backend
+
+
+JWT_EXPIRY_DEFAULT = 600
+
+
+def test_github_app_missing_parameters() -> None:
+    """Test that missing parameters raise an exception."""
+    with pytest.raises(ValueError, match=r'Missing Parameter: \[.*\]'):
+        github_app_backend()
+
+    with pytest.raises(ValueError, match=r"Missing Parameter: \['GitHub URL'\]"):
+        github_app_backend(
+            # type: ignore[arg-type]
+            app_id='123', install_id='456', private_rsa_key='key',
+        )
```

Comment:
These are clearly two tests, so they shouldn't be in the same test function. Although, it's possible to generalize them and use `@pytest.mark.parametrize` to have a single function but several tests.